core/utils/idgenerator.js

```diff
@@ -11,15 +11,16 @@
 'use strict';
 
 /**
- * @name Blockly.utils.idGenerator
- * @namespace
+ * Generators for unique IDs.
+ * @namespace Blockly.utils.idGenerator
  */
 goog.module('Blockly.utils.idGenerator');
 
 
 /**
  * Namespace object for internal implementations we want to be able to
  * stub in tests.
+ * @alias Blockly.utils.idGenerator.TEST_ONLY
```

Comment:
Consider removing this so it doesn't show up in the docs

Reply:
I've added a `@ignore` tag to prevent it from showing up in the docs.